docs/event-bus/docs/050-subscription-updates.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Subscription updates
+type: Details
+---
+
+The subscription crd can be updated using kubectl:
+
+`kubectl edit crd subscriptions.eventing.kyma.cx`
+
+Event-bus reacts on subscription crd changes, and it updates the corresponding NATS-Streaming subscription accordingly.
+
+**Note**: The current subscription update mechanism recreates a subscription with new specifications. This could lead to a scenarios where messages delivered during the recreation can get lost. An improved design that will address this concern will be provided in the future.
```

Comment:
` >**Note**: The current subscription update mechanism recreates a subscription with new specifications. This may result in the loss of messages delivered during the recreation process.  `

We typically avoid saying that the concern will be addressed in the future, because we never know when it is going to happen - that's why I would remove this line, leaving just the statement that something like this may happen. If the solution to this problem is added, we can remove the note.